RULES/core/context.go

```diff
@@ -105,6 +105,23 @@ type coverageReportInterface interface {
 	Build(ctx Context)
 }
 
+type TranslationUnit struct {
+	Source Path
+	Object OutPath
+	Flags []string
+}
+
+// AnalyzeInterface is an interface for targets cpmpatible with static analisys
+type AnalyzeInterface interface {
+	Units(ctx Context) []TranslationUnit
+	EnumerateDeps(ctx Context) []AnalyzeInterface
+}
+
+type analyzeReportInterface interface {
```

Comment:
shoudn't this be `analyzerReportInterface` of `analysisReportInterface`? Since the analysis is what produces the report and it is not the report being analyzed.